x-pack/legacy/plugins/apm/cypress/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "apm-cypress",
```

Comment:
This package doesn't get its dependencies installed by bootstrap, and having dependencies in a location like this isn't how we've done this in the past, but I'm intrigued by the idea...

Reply:
Yeah, I didn't want to bother the rest of kibana with the dependencies our e2e project needed. And I was running into a whole host of typescript issues. This way it's a completely separate project that just happens to live inside kibana. It's has the benefit of being very portable (we are not 100% sure whether it should live in kibana or somewhere else since it acts as integration test between several components)